doc-content/enterprise-only/release-notes/rn-740-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,125 @@
+[id='rn-740-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.4.0
+
+{PRODUCT} 7.4.0 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. See the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.3/html-single/release_notes_for_red_hat_decision_manager_7.3/index[_Release Notes for Red Hat Decision Manager 7.3_] for a list of {PRODUCT_DM} 7.3.0 fixed issues.
```

Comment:
Change {PRODUCT_DM} versions to 7.4 and 7.4.0.

Should the phrasing be changed to the IBM SG style. For example:
"For a list of {PRODUCT_DM} 7.4.0 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.3/html-single/release_notes_for_red_hat_decision_manager_7.4/index[_Release Notes for Red Hat Decision Manager 7.4_]"

For future RNs, is it possible to use constraints to avoid linking to the DM RNs?

Reply:
Thanks Clifton. I made the change. PM felt it was better to link than to include all fo the DM fixed issues.